src/meep_internals.hpp

```diff
@@ -153,6 +158,11 @@ void step_beta_stride1(realnum *f, component c, const realnum *g, const grid_vol
                        sigw, kapw);                                                                \
   } while (0)
 
+#define STEP_UPDATE_EDHB0(f, fc, gv, is, ie, g, u, s, chi2, chi3, fw, dsigw, sigw, kapw)           \
+  do {                                                                                             \
+    step_update_EDHB0(f, fc, gv, is, ie, g, u, s, chi2, chi3, fw, dsigw, sigw, kapw);              \
+  } while (0)
```

Comment:
Why do we need a macro here rather than just calling `step_update_EDHB0` directly?

Reply:
Actually, this line https://github.com/NanoComp/meep/blob/e4504ed58eec75eea57aebc342d5b63683c89edf/src/Makefile.am#L37
is generating a `stride1` version called (confusingly) `STEP_UPDATE_EDHB_stride10`, which we should call from the macro similar to `STEP_UPDATE_EDHB`